core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -1789,4 +1789,30 @@ public static Boolean getQueryPrefetchEnable() {
       return !prefetchEnable.equalsIgnoreCase("false");
     }
   }
+
+  /**
+   * get local dictionary size threshold in mb.
+   *
+   * @return local dictionary size threshold in mb.
+   */
+  public static int getLocalDictionarySizeThresholdInMB() {
```

Comment:
Every time parsing dictionary size property will not be good better to add while loading the carbon.properties

Reply:
done